ca/server.go

```diff
@@ -569,9 +596,13 @@ func (s *Server) UpdateRootCA(ctx context.Context, cluster *api.Cluster) {
 	// ExternalCA interface that has different implementations for
 	// different CA types. At the moment, only CFSSL is supported.
 	var cfsslURLs []string
-	for _, ca := range cluster.Spec.CAConfig.ExternalCAs {
-		if ca.Protocol == api.ExternalCA_CAProtocolCFSSL {
-			cfsslURLs = append(cfsslURLs, ca.URL)
+	for _, extCA := range cluster.Spec.CAConfig.ExternalCAs {
+		urlsCACert := extCA.CACert
+		if extCA.CACert == nil {
+			urlsCACert = rCA.CACert // if there was no CA cert (it was nil), assume it's the original cert
+		}
+		if extCA.Protocol == api.ExternalCA_CAProtocolCFSSL && bytes.Equal(urlsCACert, externalCACert) {
```

Comment:
Same thing here `rCA` is only set one right? `externalCACert` always `== rCA.CACert`